tests/test_markup.py

```diff
@@ -14,11 +18,14 @@ def test_good_markup():
 
 
 def test_bad_markup():
-    import manimpango  # noqa: F401
 
     assert not manimpango.MarkupUtils.validate(
         "<b>foo"
     ), '"<b>foo" should fail validation (unbalanced tags)'
     assert not manimpango.MarkupUtils.validate(
         "<xyz>foo</xyz>"
     ), '"<xyz>foo</xyz>" should fail validation (invalid tag)'
+
+
+def test_markup_text():
+    MarkupText("<span>Hello Manim</span>")
```

Comment:
I suggest a creating a text that actually uses some of the markup features

```suggestion
    MarkupText('<span underline="error"><b><i>Hello Manim</i></b></span>')
```